lix/packages/sdk/src/database/createSchema.ts

```diff
@@ -0,0 +1,79 @@
+import { sql, type Kysely } from "kysely";
+
+export async function createSchema(args: { db: Kysely<any> }) {
+	return await sql`
+  CREATE TABLE ref (
+    name TEXT PRIMARY KEY,
+    commit_id TEXT
+  );
+
+  CREATE TABLE file_internal (
+    id TEXT PRIMARY KEY DEFAULT (uuid_v4()),
+    path TEXT NOT NULL,
+    data BLOB NOT NULL
+  ) strict;
+
+  CREATE TABLE change_queue (
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    file_id TEXT,
+    path TEXT NOT NULL,
+    data BLOB
+  ) strict;
+
+  create view file as
```

Comment:
I extracted the schema creation from `newLixFile` to the `database` folder for co-location clarity.